### ISSUE TITLE
Missing Headings on Dashboard Page

### ISSUE THREAD

**Issue by jscholes** [benetech/MathShare#1059]
The main area of the dashboard is now split into three separate sections:

1. "Library"
2. "Recent"
3. "Pre-made Sets"

However, none of these sections have headings to facilitate convenient navigation by a screen reader user.  In addition:

* The label for the library is a graphic, rather than text.
* The text "Library" and "Recent" do not communicate any context about what they relate to.

#### Suggested Solution

Add level 2 headings for each of the sections.  The "Library" graphic can remain on-screen, but should be marked as decorative via empty alt text (`alt=""`).  The heading text should be:

1. "Problem Set Library"
2. "Recent Sets"
3. "Pre-made Sets"

In the case of the last one, it should be given a unique ID, which is then applied to the following list's opening tag as the value of an `aria-labelledby` attribute.

CC @rupeshparab

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/2889ee3b29a657878cca92bba8ce38fc86260913.
